packages/react-file-type-icons/jest.config.js

```diff
@@ -0,0 +1,3 @@
+let { createConfig } = require('@uifabric/build/jest/jest-resources');
```

Comment:
path needs to be updated
```suggestion
let { createConfig } = require('@fluentui/scripts/jest/jest-resources');
```